nasa/harmony
# DAS-1762 - Add Harmony Regridding Service to service.yml in UAT.

## Jira Issue ID

DAS-1762

## Description

This PR adds the basic plumbing for the new Harmony Regridding Service.

## Local Test Steps

* Pull this branch of the Harmony repository.
* Pull the latest `main` branch of [the Harmony Regridding Service repository](https://git.earthdata.nasa.gov/projects/SITC/repos/harmony-regridding-service/browse).
* Build the Harmony Regridding Service Docker image using the `bin/build-image` script in that repository.
* Update you local `.env` file to include `harmony-regridder` in the comma-separated list of services in `LOCALLY_DEPLOYED_SERVICES`.
* Re-bootstrap your local Harmony instance. You should see that it creates the worker and manager Docker containers for the Harmony Regridding Service.
* Issue the request below - it should give you the specified error (a validation error, for a non-geographic CRS):

Request:

```
http://localhost:3000/C1245618475-EEDTEST/ogc-api-coverages/1.0.0/collections/all/coverage/rangeset?forceAsync=true&maxResults=1&outputCrs=EPSG%3A6933
```

Expected exception:

```
Target CRS not supported: \"+proj=cea +lat_ts=30 +lon_0=0 +x_0=0 +y_0=0 +datum=WGS84 +units=m +no_defs\"
```

## PR Acceptance Checklist
* ~~Documentation updated (if needed)~~
